girlscript/girlscript/settings.py

```diff
@@ -0,0 +1,121 @@
+"""
+Django settings for girlscript project.
+
+Generated by 'django-admin startproject' using Django 2.0.5.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/2.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/2.0/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/2.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = '%&@imstkn8a#w$91i#0uitz83rkft%l*m1dfik9246^)12l_95'
```

Comment:
Why leaking the key?
Why not using env vars

Reply:
i was making a demo website so didnt really look into that

Reply:
That's a valid point. Just make sure to fix this if your website gets accepted. :)